compiler/passes/docs.cpp

```diff
@@ -350,4 +380,4 @@ std::string filenameFromMod(ModuleSymbol *mod, std::string docsWorkDir) {
   createDocsFileFolders(filename);
 
   return filename;
-}
+}
```

Comment:
Minor nit, this symbol means the `}` is the end of the file.  We prefer to have the file end with an empty line, so would like you to undo this change here.  It's a little difficult to explain.  As an example, take a look at the end of docs.h - you can go down once from the last declaration in this file, it doesn't end on the same line as the last semi-colon.  Does that make sense?

Reply:
This happens a lot for your new test files

Reply:
reverted this change